lib/takeaway.rb

```diff
@@ -0,0 +1,44 @@
+require_relative 'messager'
+require_relative 'menu'
+require_relative 'order'
+
+class Takeaway
+
+  attr_reader :basket
+
+
+  def initialize (menu, messager)
```

Comment:
Style/SpaceAfterMethodName: Do not put a space between a method name and the opening parenthesis. (https://github.com/bbatsov/ruby-style-guide#parens-no-spaces)